src/main/java/de/bonndan/nivio/input/external/RepositoryLinkHandler.java

```diff
@@ -0,0 +1,26 @@
+package de.bonndan.nivio.input.external;
+
+import de.bonndan.nivio.model.Link;
+import org.springframework.lang.NonNull;
+import org.springframework.util.StringUtils;
+
+public interface RepositoryLinkHandler {
+
+    String OPEN_ISSUES = "repo.issues.open";
+    String OPEN_PRS = "repo.pullrequests.open";
+    String README = "repo.readme";
+    String DESCRIPTION = "repo.description";
+    String ICON = "repo.icon";
+
+    @NonNull
+    default String getRepoName(Link link) {
+        try {
+            String s = link.getHref().toString().split(link.getHref().getHost())[1];
+            s = StringUtils.trimTrailingCharacter(s, '/');
+            return StringUtils.trimLeadingCharacter(s, '/');
+        } catch (Exception e) {
+            throw new RuntimeException("Failed to parse github url " + link.getHref() + ": " + e.getMessage());
```

Comment:
This default method is used both for GitHub and GitLab. I think the message should not be specific to GitHub ;)

Reply:
fixed